### PR TITLE
Eugene/wk8

### PR DESCRIPTION
**Pull request by eugenetayyj**

### CODE REVIEW

**Review by isanidiot — COMMENTED** [review]
Good job.😃  There are a lot of files to change once you add a field, and I think you covered them all!

See if you can not include all those .class files next time tho

File: src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java [hunk 1]
```diff
@@ -42,73 +45,95 @@ public void toModelType_validPersonDetails_returnsPerson() throws Exception {
...
+    }
+
+    @Test
+    public void toModelType_nullNetWorth_throwsIllegalValueException() {
+        JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_ADDRESS,
+                VALID_REMARK, null, VALID_TAGS);
+        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, NetWorth.class.getSimpleName());
+        assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
+    }
+
```
Comment (ReaganTan00) [inline src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java:129]: Good test cases for networth

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 2]
```diff
@@ -102,7 +107,19 @@ public PersonBuilder withRemark(String remark) {
...
+        return this;
+    }
+
     public Person build() {
-        return new Person(name, phone, email, address, remark, tags);
+        return new Person(name, phone, email, address, remark, netWorth, tags);
+    }
+
+    public Person buildNoRemark() {
+        return new Person(name, phone, email, address, netWorth, tags);
```
Comment (ReaganTan00) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:123]: 👍

**Review by ReaganTan00 — APPROVED** [review]
LGTM, well written for test cases

**Pull request merged**